Struggle enabling `stac`

This may be [EBKAC](https://en.wiktionary.org/wiki/EBKAC#English) (I'm still learning the new `cargo` dependency feature propagation syntax), but I'm struggling to enable the `stac` feature when using the top-level `ogcapi` crate, and wondered if you might have advice.

`Cargo.toml`:
```toml
[package]
name = "ogcapi-client-test"
version = "0.1.0"
edition = "2021"

[dependencies]
ogcapi = { git = "https://github.com/georust/ogcapi.git", features = ["client", "stac"] }
```

`src/main.rs`:
```rust
use ogcapi::client::Client;
use ogcapi::types::common::Bbox;
use ogcapi::types::stac::SearchParams;

fn main() {
    let endpoint = "https://data.geo.admin.ch/api/stac/v0.9/";
    let client = Client::new(endpoint).unwrap();
    let root = client.root().unwrap();
    println!("Root catalog id: {}", root.id);
    let catalogs = client.catalogs().unwrap();
    println!("Found {} catalogs!", catalogs.count());
    let bbox = Bbox::from([7.4473, 46.9479, 7.4475, 46.9481]);
    let params = SearchParams::new()
        .with_bbox(bbox)
        .with_collections(["ch.swisstopo.swissalti3d"].as_slice());
    let items = client.search(params).unwrap();
    println!("Found {} items!", items.count());
}
```

Output:

```cargo
❯ cargo build
   Compiling ogcapi-client-test v0.1.0 (/Users/sfitch/Coding/tmp/ogcapi-client-test)
error[E0599]: no method named `catalogs` found for struct `Client` in the current scope
  --> src/main.rs:10:27
   |
10 |     let catalogs = client.catalogs().unwrap();
   |                           ^^^^^^^^ method not found in `Client`

error[E0599]: no method named `search` found for struct `Client` in the current scope
  --> src/main.rs:16:24
   |
16 |     let items = client.search(params).unwrap();
   |                        ^^^^^^ method not found in `Client`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `ogcapi-client-test` due to 2 previous errors
```

--- a/ogcapi/Cargo.toml
+++ b/ogcapi/Cargo.toml
@@ -16,7 +16,7 @@ types = ["ogcapi-types"]
 
 import = ["gdal", "geo", "geojson", "osmpbfreader", "serde", "serde_json", "sqlx", "url", "wkb"]
 
-stac = ["ogcapi-types?/stac", "ogcapi-drivers?/stac", "ogcapi-drivers?/s3", "ogcapi-services?/stac"]
+stac = ["ogcapi-types?/stac", "ogcapi-drivers?/stac", "ogcapi-drivers?/s3", "ogcapi-services?/stac", "ogcapi-client?/stac"]
 
 [dependencies]
 anyhow = "1.0.65"

